include/model.php

```diff
@@ -367,7 +251,7 @@ public function is_translated_post_type( $post_type ) {
 	 * @param bool $filter True if we should return only valid registered taxonomies.
 	 * @return string[] Array of registered taxonomy names for which Polylang manages languages and translations.
 	 */
-	public function get_translated_taxonomies( $filter = true ) {
+	public function get_translated_taxonomies( bool $filter = true ): array {
```

Comment:
Type hinting might be risky on such public method, isn't it?

Reply:
The only time this parameter is used is in an integration test in PLL Pro.

Reply:
I meant for third party, but maybe I'm too zealous.

Reply:
Since the doc was already mentioning a boolean, I think we can keep it.

Reply:
I am thinking as Hugo.

Reply:
Reverted in [1b6f7ed](https://github.com/polylang/polylang/pull/1503/commits/1b6f7ed2f498c60d999730b027609e01bfcffd28).